### ISSUE TITLE
Bambu Studio crashes when trying to slice project

### ISSUE THREAD

**Issue by GabrielQuintana** [bambulab/BambuStudio#3318]
### Bambu Studio Version

1.8.2.56

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11 Home

### Additional system information

CPU: i9-14900k
Memory: 64 GB 6400mhz
GPU: Asus Nvidia GeForce RTX 4070 Ti

### Printer

Bambu Lab X1C

### How to reproduce

Open the provided 3mf file in bambu studios, and click "slice plate"

### Actual results

Bambu Studio crashes

### Expected results

The plate should be successful sliced

### Project file & Debug log uploads

[Log and Project.zip](https://github.com/bambulab/BambuStudio/files/13848627/Log.and.Project.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by QingZhangBambu** [comment]
We found that some models cannot be sliced on Win 11. We'll try to fix this in next big version.

**Comment by QingZhangBambu** [comment]
![image](https://github.com/bambulab/BambuStudio/assets/112676417/a40f6d9e-6d11-40ec-b869-f7a58d49ae27)

**Comment by QingZhangBambu** [comment]
@GabrielQuintana  is that still happend in the latest version on your PC?

**Comment by QingZhangBambu** [comment]
@GabrielQuintana holle, there is a method that some people said is helpful, may you try it and check if it is useful?.
For Win 11, here is how to set the affinity : 
1- Open Task Manager (CTRL+SHIFT+ESCAPE)
2- Go to Details
3- find bambu-studio.exe and right click to chose "Affinity"
![image](https://github.com/bambulab/BambuStudio/assets/112676417/57a42ead-dd48-47dd-8094-0afa42b6c41e)
4- From the popup, it is possible to choose which thread(s) to use. For example, on my i9-13900K, I've got CPU0 to CPU31 (I've 8 performance cores with 2 threads, and 16 efficiency cores -> 32 "CPUs" listed)
![image](https://github.com/bambulab/BambuStudio/assets/112676417/9f056df5-b8e9-4e83-a0e7-8c7ec6371bc6)
And interestingly enough, when I slice without setting affinity (so, Bambu Studio uses all threads available) -> it crashes.
When I first set affinity to, for example, CPU0 to CPU3, or CPU0 to CPU7 -> slicing goes through successfully.

**Comment by GabrielQuintana** [comment]
@QingZhangBambu Latest version (without modifying affinity) is still having the same issue where bambu studio crashes.

Though after setting the affinity to CPU0 to CPU7 (also tried CPU25-31) I got the exact same result, slicing does goes through successfully.

**Comment by QingZhangBambu** [comment]
@GabrielQuintana Then when the Studio crashed?

**Comment by GabrielQuintana** [comment]
@QingZhangBambu Yes, it crashes when slicing on the new version too. Should I provide logs again for the new version?

**Comment by QingZhangBambu** [comment]
yes, that would be helpful

**Comment by Blade-Strike** [comment]
I have the same exact issue. If I set infinity to 1-7 I'm good. Otherwise, it's a hard crash.
[sys.txt](https://github.com/bambulab/BambuStudio/files/14106181/sys.txt)

**Comment by QingZhangBambu** [comment]
@Blade-Strike  maybe you can do some CPU tests. some users change the CPU and all problem be solved. https://community.intel.com/t5/Processors/i9-13900K-very-frequent-crashes-Windows-11-with-apps-games-and/m-p/1528789/highlight/true
https://github.com/bambulab/BambuStudio/issues/3354

**Comment by Arcaine10** [comment]
Good morning,
I may have found a solution without having to change the affinity each time. You have to download Intel extreme tuning utility, go to advanced tuning and set performance active-core tuning to 54. Since then I have had no more problems.
![Capture d'écran 2024-03-12 072042](https://github.com/bambulab/BambuStudio/assets/163099480/951ae4a7-ef5d-468b-8b9d-4f7874d368fa)

**Comment by GabrielQuintana** [comment]
@QingZhangBambu Sorry forgot to update here, the issue was 100% the CPU being unstable because of motherboards and intel overclocking the CPU too much. which made CPU intensive procedures like slicing crash every so often. So I'm good to close this issue if you are.

For other people experiencing the same issue as I was, here's the thread that helped me https://www.reddit.com/r/overclocking/comments/1axepvu/optimizing_stability_for_intel_13900k_and_14900k/

**Comment by QingZhangBambu** [comment]
ok ;)

**Comment by elyograg** [comment]
> Good morning, I may have found a solution without having to change the affinity each time. You have to download Intel extreme tuning utility, go to advanced tuning and set performance active-core tuning to 54. Since then I have had no more problems.

This worked for me.  Slicing is now stable.  I was having a ton of problems with it.  Mostly it would just crash Bambu Studio, but sometimes it would bluescreen or freeze the system so completely that the mouse pointer wouldn't even move.

**Comment by masystems** [comment]
Got the same issue here, had to set it use CPU0-7. Bambu, please fix your software.

**Comment by elyograg** [comment]
I did have this working with the changes to the Intel Extreme Tuning program, but it has started crashing again when slicing certain models.  I didn't change anything.

Bambu Studio also crashes slicing the same model, but it causes a bluescreen instead of crashing the program.

**Comment by Skydev0h** [comment]
For me I had issue on an Intel i9-14900K, and it seems that the issue is in multithreading on both P-Cores and E-Cores.
Using a free `CoreDirector` utility I configured constraining `bambu-studio.exe` to P-Cores, and that not only fixed the crashes, but also sped up slicing by a significant margin.

![image](https://github.com/bambulab/BambuStudio/assets/5331782/03677dd4-c639-4d99-80f5-c69dca4d149d)

**Comment by evlas1013** [comment]
I'm having a similar issue but any time I go to the Preview tab, it crashes. No combination of affinity fixed it for me. i7-7700 Win10. I don't even get to the slicing step. As soon as I import a model, it crashes. If I just click the "Preview" tab, it crashes. v1.10.1.50

**Comment by QingZhangBambu** [comment]
@elyograg could you post a new issue? I will ask the concerned staff to check this. your problem is not the same as this one.

**Comment by MaJin1996** [comment]
@QingZhangBambu 
你好，最新版仍然有这个问题，环境配置和这个 issue 差不多。 通过增加 CPU 亲和性解决。
你们有考虑过换个语言写吗，看起来一年半了这个问题仍然存在，可以预见这个问题以后也不会得到彻底解决。

**Comment by MaJin1996** [comment]
看了下这问题都一年半了，还没解决，高配电脑等于是不能用你们的软件，你们能不能招点正经C++ 工程师或者干脆别用C++写了。
